students/shibin_mathew/lesson2/FizzBuzz.py

```diff
@@ -0,0 +1,14 @@
+def Fizz(number):
```

Comment:
per python style guidelines function names should be lower case snake case, examples:
`def fizz(number)`
`def fizz_buzz(number)`